src/gui/advancedsettings.cpp

```diff
@@ -723,6 +727,9 @@ void AdvancedSettings::loadAdvancedSettings()
     // Recheck completed torrents
     m_checkBoxRecheckCompleted.setChecked(pref->recheckTorrentsOnCompletion());
     addRow(RECHECK_COMPLETED, tr("Recheck torrents on completion"), &m_checkBoxRecheckCompleted);
+    // Custom application title
+    m_lineEditCustomAppTitle.setText(pref->appTitle());
+    addRow(CUSTOM_APP_TITLE, tr("Custom application title"), &m_lineEditCustomAppTitle);
```

Comment:
It doesn't change the title, it adds a suffix to it. 
Also i don't know if it matters but "Allow to set custom text to main window title" is not the whole truth. The suffix is added to all dialogs.

Reply:
You are right. I'll rename the code and the commit.

Reply:
>The suffix is added to all dialogs.

I suppose this is not the intended behavior. I'll need revise the implementation.